images/openshift-ci/google-cloud-sdk.repo

```diff
@@ -0,0 +1,8 @@
+[google-cloud-sdk]
+name=Google Cloud SDK
+baseurl=https://packages.cloud.google.com/yum/repos/cloud-sdk-el7-x86_64
```

Comment:
the reason to base on EL7 ?

Reply:
we do have https://packages.cloud.google.com/yum/repos/cloud-sdk-el8-x86_64 one but we do need to try that.